### ISSUE TITLE
Vertical facets options doesn't work

### ISSUE THREAD

**Issue by dannyparsons** [africanmathsinitiative/R-Instat#4424]

**Comment by dannyparsons** [comment]
This could have been because two facet variables were used. If so, then if `dir` isn't a parameter of `facet_grid()` so vertical/horizontal options shouldn't have been visible.

**Comment by maxwellfundi** [comment]
@dannyparsons  I have looked at this and this is the missing detail i have found out

There is something we need to change here.
1. We use `facet_wrap()` when we are faceting with only one variable and use` facet_grip()` when we are faceting with two variables. 
We use the Horizontal or Vertical facets when using the `facet_wrap()` when we have none of the checkboxes checked but when we check either of the checkboxes, then we use `facet_grid()` . Its in this case then we need the two radio buttons for the Horizontal and vertical not to be visible as well as when we have two variables filled in.

**Comment by maxwellfundi** [comment]
This is moved from #4514 
A very small issue I hope.

On the facets tab (sub-dialogue for the graphs), the "Free Y axis" option had been ticked. It wasn't needed so we de-selected the option. And the resulting graph was OK. But coming back to that TAB it has re-selected itself.
I assume this may be the case with other options also, at least on this Facets TAB, but I haven't checked.

**Comment by maxwellfundi** [comment]
@volloholic  now suggests that we change this tab. The change would be that we have a row and column facet variables instead of the current 1st facet and 2nd facet and this way, we would have both facets variables as optional.

**Comment by rdstern** [comment]
Where is this change?  I often use a single facet, particularly year or month, and now possibly quarter.  When one facet is used, e.g. month, then I hope it will, by default, have the layout as a rectangle and not 12 in a line (or column).  Or have I misunderstood this possible change?

**Comment by maxwellfundi** [comment]
@rdstern  When using  `facet_grid` facetting by one variable would mean we have the 12 in a line or column depending with if you selected the row or column variable. 
I am not sure if that is what would be desirable because @volloholic  was for the idea we get rid with the panel and the checkboxes for the number of columns and rows when we use `facet_wrap`

**Comment by rdstern** [comment]
I agree that looks OK, but I assume that I can also ask for 3 by 4 instead?  I might have dekads, e.g. 36 instead of months, and then I guess that 6 by 6 will be clearer than 36 by 1?

**Comment by dannyparsons** [comment]
We can fix the number of rows/columns for facet_wrap to 1 to get something similar to what facet_grid does, but I agree it would be good allow the user to choose to do facet_grid with one variable - but I am not sure how that would look on the dialog.
